docs/configuration/overview.md

```diff
@@ -15,7 +15,10 @@ There are three configuration files for Yazi:
 
 You can find the default configuration files at https://github.com/sxyazi/yazi/tree/main/yazi-config/preset.
 
-To override any of the defaults, begin by copying the appropriate file from the directory linked above to the Yazi configuration directory. On Unix/Unix-like systems, the configuration directory is `~/.config/yazi/`, and on Windows, the configuration directory is `C:\Users\USERNAME\AppData\Roaming\yazi\config\` (replacing `USERNAME` with your username).
+To override any of the defaults, begin by copying the appropriate file (from the directory linked above) to either:
+
+- `~/.config/yazi/` on Unix/Unix-like systems.
+- `C:\Users\USERNAME\AppData\Roaming\yazi\config\` on Windows.
```

Comment:
Thoughts on this now?

Reply:
Maybe we can place the platform name at the front, like `Unix: ~/.config/yazi`?

I think by doing so, it would be easy to distinguish them just by looking at the first few characters, thus saving time.

Reply:
I've never seen any other documentation do that, I feel like this is more clean and neat lol.

Reply:
Alright this is the last remaining thing on this PR. I'll revert as you requested.